src/mockgoose.ts

```diff
@@ -51,6 +51,21 @@ export class Mockgoose {
       });
     });
   }
+
+  shutdown(): Promise<number> {
```

Comment:
Maybe you should add too
```
this.mongooseObj.mocked = false;
```

Unmock the passed mongoose object. This will be useful in case the `mongoose` object you have used in your test in unique for all of them.